eng/pipelines/inventory.yml

```diff
@@ -0,0 +1,44 @@
+# Inventory PowerShell modules installed
```

Comment:
Can you name this a little less generic something like agent-software-inventory.yml